Exercises.Linq/TaskData.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+
+namespace Exercises.Linq
+{
+    public class TaskData
+    {
+        public List<string> Fruits { get; } = new List<string>()
+            {
+                "Lemon", "Apple", "Orange", "Lime", "Watermelon", "Loganberry"
+            };
+
+        public List<int> MixedNumbers { get; } = new List<int>() { 15, 8, 21, 24, 32, 13, 30, 12, 7, 54, 48, 4, 49, 96 };
+
+        public List<string> Names { get; } = new List<string>()
+            {
+                "Heather", "James", "Xavier", "Michelle", "Brian", "Nina",
+                "Kathleen", "Sophia", "Amir", "Douglas", "Zarley", "Beatrice",
+                "Theodora", "William", "Svetlana", "Charisse", "Yolanda",
+                "Gregorio", "Jean-Paul", "Evangelina", "Viktor", "Jacqueline",
+                "Francisco", "Tre"
+            };
+
+        public List<double> Purchases { get; } = new List<double>()
+            {
+                2340.29, 745.31, 21.76, 34.03, 4786.45, 879.45, 9442.85, 2454.63, 45.65
+            };
+
+        public List<int> wheresSquaredo = new List<int>()
+            {
+                66, 12, 8, 27, 82, 34, 7, 50, 19, 46, 81, 23, 30, 4, 68, 14
+            };
+
+        public List<Customer> Customers { get; } = new List<Customer>() {
+        new Customer(){ Name="Bob Lesman", Balance=80_345.66, Bank="FTB"}, 
```

Comment:
Please wrap all `=` with spaces. Make sure all `{` has space after it and `}` has space before it:
```csharp
{ Name = "Bob", Value = 1 }
```